tox.ini

```diff
@@ -17,7 +17,7 @@ commands =
     django_not_installed: bash -c \'pylint --load-plugins=pylint_django setup.py | grep django-not-available\'
     django_is_installed: pylint --load-plugins=pylint_django --disable=E5110 setup.py
     flake8: flake8
-    pylint: pylint --rcfile=tox.ini -d missing-docstring,too-many-branches,too-many-return-statements,too-many-ancestors,fixme --ignore=tests pylint_django setup
+    pylint: pylint --rcfile=tox.ini -d missing-docstring,too-many-branches,too-many-return-statements,too-many-ancestors,fixme,duplicate-code --ignore=tests pylint_django setup
```

Comment:
FTR I find duplicate-code checker to be too relaxed. It will match any 3 lines which are the same/similar and will produce lots of false positives. +1 for disabling.

Reply:
Yeah the issue here was with imports in two checkers being the same in the same order, there's no way round that (some base class where you import * from the parent is also bad) so it was a bit annoying as `# pylint: disable=` didn't work either. Shame but never mind.